src/templates/contest/contest.html

```diff
@@ -45,7 +45,7 @@ <h1 style="display: inline;">{{ title }}</h1>
                 </td>
                 <td><a href="{{ request.path }}/problem/{{ row["problem_id"] }}">{{ row["name"] }}</a></td>
                 <td>{{ row["category"] }}</td>
-                <td>{{ row["point_value"] }}</td>
+                <td>{{ row["point_value"] }}{{ "D" if row["dynamic"] }}</td>
```

Comment:
I think we should make this say "Dynamic" instead or something more descriptive

Reply:
Perhaps "This problem is dynamically scored" if someone hovers over it?